.ci/azure/windows.yml

```diff
@@ -23,8 +23,6 @@ jobs:
 - job: Win
   strategy:
```

Comment:
whole matrix section can be removed, moving CMAKE_BUILD_SHARED_LIBS to variables

Reply:
kept as is to get an ability to check it manually via PR (I mean Windows dynamic libraries)